apps/evm_family/evm_pub_key.c

```diff
@@ -345,28 +402,42 @@ void evm_get_pub_keys(evm_query_t *query) {
                              &seed[0])) {
     memzero(seed, sizeof(seed));
     // TODO: Handle error case reporting to host
+    memzero(seed, sizeof(seed));
     return;
   }
 
   delay_scr_init(ui_text_processing, DELAY_SHORT);
 
-  bool status = get_public_keys(init_req->derivation_paths,
-                                init_req->derivation_paths_count,
-                                seed,
-                                public_keys);
+  if (!fill_public_keys(init_req->derivation_paths,
+                        seed,
+                        public_keys,
+                        init_req->derivation_paths_count)) {
+    // Clear seed as soon as it is not needed
+    memzero(seed, sizeof(seed));
```

Comment:
Seed will not be cleared in happy flow, please re-check

Reply:
fixed